server/controllers/chats.js

```diff
@@ -27,7 +27,7 @@ module.exports = {
       .findAll({
         include: {
           model: User,
-          attributes: ['name', 'imageUrl', 'email']
+          attributes: ['username', 'imageUrl', 'email']
```

Comment:
Can we declare this somewhere and re-use instead of repeating the same things everywhere.